integrations/octopus/.port/resources/blueprints.json

```diff
@@ -0,0 +1,292 @@
+          "description": "The project description"
+        },
+        "spaceId": {
+          "type": "string",
+          "title": "Space ID",
+          "description": "The ID of the space in which the project exists"
+        },
+        "isDisabled": {
+          "type": "boolean",
+          "title": "Is Disabled",
+          "description": "Indicates if the project is disabled"
+        },
+        "lifecycleId": {
+          "type": "string",
+          "title": "Lifecycle ID",
+          "description": "The lifecycle ID associated with the project"
+        },
+        "tenantedDeploymentMode": {
+          "type": "string",
+          "title": "Tenanted Deployment Mode",
+          "description": "The deployment mode regarding tenants"
+        },
+        "allowIgnoreChannelRules": {
+          "type": "boolean",
+          "title": "Allow Ignore Channel Rules",
+          "description": "Indicates if channel rules can be ignored"
+        }
+      }
+    },
+    "calculationProperties": {}
+  },
+  {
+    "identifier": "octopusDeployRelease",
+    "title": "Octopus Deploy Release",
+    "icon": "Octopus",
+    "description": "A release in Octopus Deploy",
+    "schema": {
+      "properties": {
+        "version": {
+          "type": "string",
+          "title": "Version",
+          "description": "The version of the release"
+        },
+        "assembled": {
+          "type": "string",
+          "title": "Assembled",
+          "format": "date-time",
+          "description": "The datetime the release was assembled"
+        },
+        "channelId": {
+          "type": "string",
+          "title": "Channel ID",
+          "description": "The ID of the channel associated with the release"
+        },
+        "releaseNotes": {
+          "type": "string",
+          "title": "Release Notes",
+          "description": "Notes provided for the release"
+        },
+        "projectId": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The ID of the project this release belongs to"
+        },
+        "url": {
+          "type": "string",
+          "title": "Release URL",
+          "format": "url",
+          "description": "The URL to access the release in Octopus Deploy"
+        }
+      }
+    },
+    "calculationProperties": {},
+    "relations": {
+      "project": {
+        "target": "octopusDeployProject",
+        "title": "Project",
+        "description": "The project to which this release belongs",
+        "required": false,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "octopusDeployDeployment",
+    "title": "Octopus Deploy Deployment",
+    "icon": "Octopus",
+    "description": "A deployment in Octopus Deploy",
+    "schema": {
+      "properties": {
+        "releaseId": {
+          "type": "string",
+          "title": "Release ID",
+          "description": "The ID of the release being deployed"
+        },

```

Comment:
this should be a relation to the release blueprint

Reply:
Updated